documentation/modules/auxiliary/scanner/mssql/mssql_login.md

```diff
@@ -0,0 +1,43 @@
+## Vulnerable Application
+
+This module simply queries the MSSQL instance for a specific user/pass (default is sa with blank). 
+
```

Comment:
Inside the vulnerable application section it's always important (if possible) to show the user how they can setup a target in order to test the module.
```suggestion

### Setup
A docker container can be spun up with the following command to test this module:
`docker run -e 'ACCEPT_EULA=Y' -e 'MSSQL_SA_PASSWORD=N0tpassword!' -p 1433:1433 -d mcr.microsoft.com/mssql/server:2022-latest`

```